compiler/test/formatter_outputs/infix.gr

```diff
@@ -4,7 +4,7 @@ let trimString = (str: String, end: Bool) => {
 
   for (; i < charsLength && i > -1; i += offset) {
     let currentChar = chars[i]
-    // TODO: Use unicode whitespace property and unicode line terminator once github issue #661 is completed
+    // TODO(#661): Use unicode whitespace property and unicode line terminator once github issue 661 is completed
```

Comment:
I think this can actually be removed because it was probably a copy-paste of some code in the stdlib into the formatter inputs.
```suggestion
```